exporter/kineticaexporter/logs_exporter.go

```diff
@@ -0,0 +1,55 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kineticaexporter
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+)
+
+type kineticaLogsExporter struct {
+	logger *zap.Logger
+}
+
+// newLogsExporter
+//
+//	@param logger
+//	@param cfg
+//	@return *kineticaLogsExporter
+//	@return error
+func newLogsExporter(logger *zap.Logger, cfg *Config) (*kineticaLogsExporter, error) {
+	if err := cfg.Validate(); err != nil {
```

Comment:
We shouldn't need to call validate ourselves, the collector's framework will handle that for us. This applies to all signals.

Reply:
Removed this check from all signals.